tests/drivers/sensor/src/dummy_sensor.c

```diff
@@ -111,6 +111,20 @@ int dummy_sensor_attr_set(struct device *dev,
 	return 0;
 }
 
+int dummy_sensor_attr_get(struct device *dev,
+		      enum sensor_channel chan,
+		      enum sensor_attribute attr,
```

Comment:
It would probably be nice to use `attr` here.

Reply:
Agreed. I just reversed what `dummy_sensor_attr_set()` does (which does not use `attr`). I will implement a check for `attr` in both functions. Thank you for reviewing.

Reply:
@gmarull Please re-review.